quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/SelectiveMigratorFunction.java

```diff
@@ -182,15 +183,25 @@ private static MigratorFunction createInsertMigrator(NullRecords nullRecords, Re
 				.map(column -> functionParameterMapping.get(column.getName()) + " " + column.getType().toString())
 				.collect(Collectors.toList());
 
-		TableRef sourceRef = refLog.getTableRefById(source.getName());
```

Comment:
If a table has been renamed this function will get the old one. The column mapping function should still work from this tableref but it does not...